src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesObsolete.TableExt.al

```diff
@@ -12,6 +12,14 @@ tableextension 308 NoSeriesObsolete extends "No. Series"
 {
     fields
     {
+        field(11790; Mask; Text[20]) // CZ Functionality
```

Comment:
Is there a need to move this when it is already removed?

Reply:
yes... putting all obsolete stuff in this table extension so we get clean code in the base table